src/commands/deploy.js

```diff
@@ -366,6 +368,26 @@ const printResults = ({ flags, results, deployToProduction, exit }) => {
   }
 }
 
+const hideRedirects = async (deployFolder) => {
+  await renameAsyncNoError(`${deployFolder}/_redirects`, `${deployFolder}/.redirects`)
+}
+
+const unhideRedirects = async (deployFolder) => {
+  await renameAsyncNoError(`${deployFolder}/.redirects`, `${deployFolder}/_redirects`)
+}
+
+const syncRedirects = async ({ oldConfig, newConfig, deployFolder, configPath }) => {
+  if (oldConfig.redirects === newConfig.redirects) {
```

Comment:
I've tested it and this evaluates to `true` is a plugin didn't change the redirects.
We could use `fast-deep-equal` too but since customers can have many redirects we might want to avoid it.